dlt/cli/magics.py

```diff
@@ -0,0 +1,288 @@
+from argparse import ArgumentParser
+
+# from dlt.cli._dlt import telemetry_change_status_command_wrapper, init_command_wrapper, on_exception
+# from .deploy_command import COMMAND_DEPLOY_REPO_LOCATION, DLT_DEPLOY_DOCS_URL
+
+# from dlt.cli._dlt import init_command_wrapper
+# from .pipeline_command import DLT_PIPELINE_COMMAND_DOCS_URL
+# from dlt.cli.init_command import init_command
+
+from IPython.core.display import HTML, display # type: ignore
+from IPython.core.magic import (
+    Magics,
+    cell_magic,
+    line_cell_magic,
+    line_magic,
+    magics_class, register_line_magic,
+) # type: ignore
+from IPython.core.magic_arguments import argument, magic_arguments, parse_argstring # type: ignore
+
+
+from ..common.runtime.exec_info import is_notebook, is_ipython
+from dlt.cli import echo as fmt
+
+
+import typing as t  # type: ignore
+from hyperscript import h  # type: ignore
+import os
+import traceback
+
+import click
+
+DEBUG_FLAG = False
+
+DLT_INIT_DOCS_URL = "https://dlthub.com/docs/reference/command-line-interface#dlt-init"
+DEFAULT_VERIFIED_SOURCES_REPO = "https://github.com/dlt-hub/verified-sources.git"
+DLT_DEPLOY_DOCS_URL = "https://dlthub.com/docs/walkthroughs/deploy-a-pipeline"
+
+
+
+@magics_class
+class DltMagics(Magics):
+
+    @property
+    def display(self) ->  t.Any:
+        if os.getenv("DATABRICKS_RUNTIME_VERSION"):
+            # Assume Databricks' 'display' is a callable with an unknown signature
+            databricks_display = self._shell.user_ns.get("display")
+            if callable(databricks_display):
+                return databricks_display
+            else:
+                raise RuntimeError("Expected a callable for Databricks' display, got something else.")
+        return display  # Assuming 'display' is a predefined callable
+
+    def success_message(self, messages: t.Dict[str, str]) -> HTML:
+        unstyled = messages.get("unstyled")
```

Comment:
hmmm interesting. does our `fmt.bold` etc. still works or we should patch it with magic-specific verions?

Reply:
It does. This is more pretty, can go back to defaults or move this there. wdyt?

Reply:
keep it here if it is prettier :)